data/config/default/mermaid.json

```diff
@@ -0,0 +1,6 @@
+{
+  "library": {
+    "path": "https://cdnjs.cloudflare.com/ajax/libs/mermaid/8.0.0/mermaid.min.js"
+  },
+  "theme": "default"
```

Comment:
Replace `theme` with an empty `config` object like this:

```json
{
  "library": {
    "path": "//unpkg.com/mermaid/dist/mermaid.min.js"
  },
  "config": {}
}
```

`default` theme will automatically be used.